tools/ci-trigger/pr_test.go

```diff
@@ -234,3 +236,72 @@ func TestPopulateTestDetail(t *testing.T) {
 		t.Errorf("populateModifiedTests(%v): -want,+got:\n%s", modifiedTests, diff)
 	}
 }
+
+func TestWithRetry(t *testing.T) {
+	var retryCount int
```

Comment:
I noticed the language alternating between "retry" and "attempt." I'm not sure if you had a particular reason why you one over another in the various in the code and test, but maybe something to consider. I don't have a strong opinion either way, although some seem to "feel" right for reasons I can't articulate yet.

Reply:
That one variable name did make it inconsistent.  Each individual retry is an attempt, but the overall operation is a retry.  Hopefully it makes more sense now.